### PR TITLE
Update sponsorship to include blub

### PR DESCRIPTION
**Pull request by JDurstberger**

### CODE REVIEW

File: sponsorship.md [hunk 1]
```diff
@@ -12,6 +12,18 @@ SoCraTes UK is a non-profit conference organised by and for the Software Crafter
 our attendees only need to pay for their hotel room. As such, we rely heavily on our sponsors to reduce 
 the overall cost of attendance and in exchange, we provide exposure (and well deserved Kudos!) to all of our sponsors.
 
+<br>
+
+### Why Sponsor SoCraTes UK?
```
Comment (jukecraft) [inline sponsorship.md]:
```suggestion
### Why sponsor SoCraTes UK?
```

File: sponsorship.md [hunk 2]
```diff
@@ -12,6 +12,18 @@ SoCraTes UK is a non-profit conference organised by and for the Software Crafter
 our attendees only need to pay for their hotel room. As such, we rely heavily on our sponsors to reduce 
 the overall cost of attendance and in exchange, we provide exposure (and well deserved Kudos!) to all of our sponsors.
 
+<br>
```
Comment (jukecraft) [inline sponsorship.md:15]: Not sure those new lines are necessary?

File: sponsorship.md [hunk 3]
```diff
@@ -12,6 +12,18 @@ SoCraTes UK is a non-profit conference organised by and for the Software Crafter
...
+
+### Why Sponsor SoCraTes UK?
+
+SoCraTes brings together a community of software professionals who care about creating high-quality, maintainable software, and want to work together to share and build on their skills.  
+As an unconference with no predetermined agenda, SoCraTes is created by its participants. As a result, it attracts highly engaged, collaborative practitioners, and encourages dense networks of ideas.  
+By sponsoring SoCraTes, you can demonstrate that you share these values of collaboration and continuous improvement in software development. If you are recruiting, then this is a great way to reach out to a community of highly engaged potential candidates.  
+
+<br>
```
Comment (jukecraft) [inline sponsorship.md:23]: Not sure those new lines are necessary?

File: sponsorship.md (hunk 2 above)
Comment (JDurstberger) [reply sponsorship.md:15]: We have them on all pages. 
I had a look what it looks like without them when we did the bigger changes on the website and it becomes incredibly vertical dense. 

It is definitely easier to read when we have them in

File: sponsorship.md (hunk 3 above)
Comment (JDurstberger) [reply sponsorship.md:23]: See https://github.com/lscc/socrates-uk/pull/74#discussion_r265917805

**Review by jukecraft — APPROVED** [review]

**Pull request merged**